### ISSUE TITLE
Horrible view of the Android application icons

### ISSUE THREAD

**Issue by SmithJohnFirst** [godotengine/godot#34816]
**Godot version:**
3.1

**OS/device including version:**
Android 9

**Issue description:**
All my Android apps have changed the icon. Applications developed using Godot. The icons are terrible - a white frame has appeared around the icon.
![0](https://user-images.githubusercontent.com/43090905/71765001-5f19ed80-2f08-11ea-82b3-c86ab729e877.png)
Previously, on older versions of Android, they looked great.
I did not change the icon images.

**Comment by KoBeWi** [comment]
I don't think it's Godot's problem. The white frames appeared after Android update, so you probably need to update your icons to meet some new guidelines.

**Comment by timothyqiu** [comment]
FYI: This is the result of [Adaptive Icon](https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive) introduced since Android O (API level > 26).

To get rid of the frame, the application's `android:icon` attribute in AndroidManifest.xml has to point to an Adaptive Icon drawable XML.

It seems that there is no mention of adaptive icon in the codebase.

**Comment by SmithJohnFirst** [comment]
My applications do not use adaptive icons.
I decided to add them, but did not find the corresponding menu item in the project export.
![01](https://user-images.githubusercontent.com/43090905/71766580-ce98d880-2f1a-11ea-88be-48fe7904cc96.PNG)
I looked at the contents of the export templates and did not find adaptive icons.
![02](https://user-images.githubusercontent.com/43090905/71766611-27687100-2f1b-11ea-9ebd-ede2736eb2ca.PNG)
Also, I did not find mention of adaptive icons in the documentation. As I understand it, Godot does not support this feature.
